src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -116,6 +117,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         getAnnotationsPath: noop,
         hasReplies: false,
         hasTasks: true,
+        hasTitle: true,
```

Comment:
I don't think we want to expose this option at the top-level. In this case, I'd suggest using the `features` prop via `isFeatureEnabled`, such as `isFeatureEnabled('activityFeed.filter.enabled')`

Reply:
I think that activity feed filter is not connected directly to hiding the title. I can imagine that in future when threads will be introduced in Preview title should still be visible alongside the filter. Or do you think those two (title and filter) are mutually exclusive?

I'm implementing the filter itself in a different branch and will be utilizing `features` functionality to control it.

Reply:
As agreed I've removed `hasTitle` prop and title visibility will be controlled with `activityFeed.filter` feature (implemented in separate branch).